spec/support/login_helper.rb

```diff
@@ -0,0 +1,10 @@
+module LoginHelper
+  def log_in(user)
+    visit new_user_session_path
+
+    fill_in 'Email', with: user.email
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.